docs/architecture/3rd party technical review/000-technical-review-records.md

```diff
@@ -0,0 +1,43 @@
+# 3rd Party Technical Review Format
+
+## Context
+
+We are using succefully lightweight architecture decision records to document decision being made about architecture.
+Last year legislation changes around data privacy (GDPR, CCPA, PECR) and their impact on our obligations means we need to  increase our technical measures to ensure the safety of our readers data through:
+  * Reducing number of 3rd parties    
+  * Architecting our systems with a [privacy by design](https://en.wikipedia.org/wiki/Privacy_by_design) approach 
+  * Documenting risks and mitigations in place or to be added.
+  * Adding new controls to prevent issues, i.e being proactive rather than reactive.
+
+
+Additionally to this privacy concern, we have noticed in the past the difficulty to correctly estimate and measure accurately the performance andm  by association our revenue impact of 3rd party. While they usually come with a direct revenue, there is as well indirect revenue loss through performamce impact:
+ * performance degradation impacts our ad display and our advertising revenue  
+ * performance degradation impacts our contributions conversion rate 
+ * performance degradation impacts our Google SEO which impact our reach and indirectly our overall revenue
+
+
+To address those 2 concerns, we are making a change to the process for deciding if a 3rd party will be added. 
+Before a decision is made by the business to add a third party to our website or apps, we will perform an engineering technical review that will be used to inform the business decision.
+
+The format of the 
+
+* Format of the engineering technical review:
+ * Metadata about the review
+   * Date of the review
+   * Author of the review
+   * Platforms concerned
+ * Context
+ * Data privacy engineering review
+ * Security engineering review
+ * Perfomance engineering review
+ * Recommended mitigations
+
+## Decision
+
+For all 3rd party technical reviews in this project, a record will be created with a filename of `XXX-technical-review.md` where XXX is the monotonically increasing number described in the linked document.
+
+Pleae note that as records will be public, specific attention should be paid about information being disclosed.
+
+## Status
+
+Approved
```

Comment:
@jfsoul @nicl @gtrufitt I am documenting here some business process change regarding adding third-party scripts to our website. I think using `architecture` doc and formalism is a great way to document further technical reviews but let me know if you disagree.

Reply:
This sounds good to me although we may want to also think about how we document (or carry out even) these reviews when they cover multiple platforms outside DCR (e.g. apps)